recipes/limix-legacy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "limix-legacy" %}
+{% set version = "0.8.10" %}
+{% set sha256 = "9e9e5a66283272536001d8f6fc90c4b36ae9d9b0cc34a63017549f76050eecac" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy >=1.10
+  run:
+    - python
+    - scikit-learn
+    - pandas
+    - numpy >=1.10
+    - scipy
+    - h5py
+    - matplotlib
+    - numpy-sugar >=1.0.34
+    - dask >=0.14
+    - pandas-plink >=1.1.6
+    - pytables
+
+test:
+  requires:
+    - pytest >=3
+  imports:
+    - limix_legacy
+  commands:
```

Comment:
It looks like there are a bunch of command line entry points that should be tested:

https://github.com/limix/limix-legacy/blob/master/setup.py#L199-L208

and included in the top level `build` section:

https://conda.io/docs/building/meta-yaml.html#python-entry-points